deploy/osd-17042-cdo-olm-dance/cronjob.yaml

```diff
@@ -88,14 +91,15 @@ spec:
               #!/bin/bash
               set -euxo pipefail
               NAMESPACE=openshift-custom-domains-operator
-              oc -n "$NAMESPACE" 
-              # Just to be paranoid, let's only do this if we find the subscription there to begin with
-              COUNT_SUB=$(oc -n "$NAMESPACE" get subscriptions.operators.coreos.com custom-domains-operator -o name | wc -l) || true
-              if [[ $COUNT_SUB -ge 1 ]]; then
-                oc patch subscriptions.operators.coreos.com -n "$NAMESPACE" custom-domains-operator --type json --patch='[ { "op": "remove", "path": "/metadata/annotations/kubectl.kubernetes.io~1last-applied-configuration" } ]'
+              # Check for the presence of CDO v0.1.201, then we know we're stuck
+              # stuck
+              CSV_FOUND=$(oc -n "$NAMESPACE" get clusterserviceversions.operators.coreos.com custom-domains-operator.v0.1.201-b9b8893 -o name --ignore-not-found) || true
+              if [[ ! -z "$CSV_FOUND" ]]; then
+                # It is present, so wipe out CDO
+                oc -n "$NAMESPACE" delete clusterserviceversions.operators.coreos.com -l operators.coreos.com/custom-domains-operator.openshift-custom-domains-operator=""
+                oc -n "$NAMESPACE" delete installplan.operators.coreos.com --all
```

Comment:
We want to only delete installplans for CDO 

`oc get installplans -n openshift-custom-domains-operator -l operators.coreos.com/custom-domains-operator.openshift-custom-domains-operator=""`